src/cli.js

```diff
@@ -68,7 +80,7 @@ const getPlugins = config => Object.keys(config.plugins || {})
 const config = cfgResolve(cli);
 
 const processing = async file => {
-  const output = await outResolve(file, config.output);
+  const output = await outResolve(file, config.output, config.root, config.allInOutput);
```

Comment:
I suggest that you pass all the settings here, and below you can take the necessary destructurization

```diff
- const output = await outResolve(file, config.output, config.root, config.allInOutput);
+ const output = await outResolve(file, config);
```